plugins/woocommerce-admin/client/core-profiler/index.tsx

```diff
@@ -1107,6 +1115,11 @@ export const coreProfilerStateMachineDefinition = createMachine( {
 							],
 							target: 'pluginsSkipped',
 						},
+						PLUGINS_LEARN_MORE_LINK_CLICKED: {
+							actions: [
+								'recordTracksPluginsLearnMoreLinkClicked',
```

Comment:
It seems `recordTracksPluginsLearnMoreLinkClicked` isn't defined anywhere? 🤔

Reply:
🤦 I think I missed it from the last rebase.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/38616/commits/31f8600ae5bc465e5481b836fe9b1ec2bbc8a3fb